packages/editor-ui/src/views/SetupView.vue

```diff
@@ -1,129 +1,123 @@
-			title: this.$locale.baseText('auth.setup.setupOwner'),
-			buttonText: this.$locale.baseText('auth.setup.next'),
-			inputs: [
-				{
-					name: 'email',
-					properties: {
-						label: this.$locale.baseText('auth.email'),
-						type: 'email',
-						required: true,
-						validationRules: [{ name: 'VALID_EMAIL' }],
-						autocomplete: 'email',
-						capitalize: true,
-					},
-				},
-				{
-					name: 'firstName',
-					properties: {
-						label: this.$locale.baseText('auth.firstName'),
-						maxlength: 32,
-						required: true,
-						autocomplete: 'given-name',
-						capitalize: true,
-					},
-				},
-				{
-					name: 'lastName',
-					properties: {
-						label: this.$locale.baseText('auth.lastName'),
-						maxlength: 32,
-						required: true,
-						autocomplete: 'family-name',
-						capitalize: true,
-					},
-				},
-				{
-					name: 'password',
-					properties: {
-						label: this.$locale.baseText('auth.password'),
-						type: 'password',
-						required: true,
-						validationRules: [{ name: 'DEFAULT_PASSWORD_RULES' }],
-						infoText: this.$locale.baseText('auth.defaultPasswordRequirements'),
-						autocomplete: 'new-password',
-						capitalize: true,
-					},
-				},
-				{
-					name: 'agree',
-					properties: {
-						label: this.$locale.baseText('auth.agreement.label'),
-						type: 'checkbox',
-					},
-				},
-			],
-		};
+import type { IFormBoxConfig } from '@/Interface';
+import { MORE_ONBOARDING_OPTIONS_EXPERIMENT, VIEWS } from '@/constants';
 
-		return {
-			FORM_CONFIG,
-			loading: false,
-		};
-	},
-	computed: {
-		...mapStores(useSettingsStore, useUIStore, useUsersStore, usePostHog),
-	},
-	methods: {
-		async onSubmit(values: { [key: string]: string | boolean }) {
-			try {
-				const forceRedirectedHere = this.settingsStore.showSetupPage;
-				const isPartOfOnboardingExperiment =
-					this.posthogStore.getVariant(MORE_ONBOARDING_OPTIONS_EXPERIMENT.name) ===
-					MORE_ONBOARDING_OPTIONS_EXPERIMENT.variant;
-				this.loading = true;
-				await this.usersStore.createOwner(
-					values as { firstName: string; lastName: string; email: string; password: string },
-				);
+import AuthView from '@/views/AuthView.vue';
 
-				if (values.agree === true) {
-					try {
-						await this.uiStore.submitContactEmail(values.email.toString(), values.agree);
-					} catch {}
-				}
+const toast = useToast();
+const locale = useI18n();
+const router = useRouter();
 
-				if (forceRedirectedHere) {
-					if (isPartOfOnboardingExperiment) {
-						await this.$router.push({ name: VIEWS.WORKFLOWS });
-					} else {
-						await this.$router.push({ name: VIEWS.NEW_WORKFLOW });
-					}
-				} else {
-					await this.$router.push({ name: VIEWS.USERS_SETTINGS });
-				}
-			} catch (error) {
-				this.showError(error, this.$locale.baseText('auth.setup.settingUpOwnerError'));
-			}
-			this.loading = false;
+const posthogStore = usePostHog();
+const settingsStore = useSettingsStore();
+const uiStore = useUIStore();
```

Comment:
nitpick: follow order :
1. Props and Emits
2. Stores
3. Composables
4. Data
5. Computed
6. Methods
7. Watchers
8. Lifecycle